server/routes/stats_router.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ResponseError } from '@opensearch-project/opensearch/lib/errors';
+import { Logger } from '@osd/logging';
+import {
+  IOpenSearchDashboardsResponse,
+  IRouter,
+  SavedObjectsFindResponse,
+} from '../../../../src/core/server';
+import { APP_API, APP_PATH, MAP_SAVED_OBJECT_TYPE } from '../../common';
+import { getStats } from '../common/stats/stats_helper';
+import { MapSavedObjectAttributes } from '../../common/map_saved_object_attributes';
+
+export function statsRoute(router: IRouter, logger: Logger) {
+  router.get(
+    {
+      path: `${APP_API}${APP_PATH.STATS}`,
+      validate: {},
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      try {
+        const savedObjectsClient = context.core.savedObjects.client;
+        const mapsSavedObjects: SavedObjectsFindResponse<MapSavedObjectAttributes> =
+          await savedObjectsClient?.find({ type: MAP_SAVED_OBJECT_TYPE, perPage: 1000, page: 1 });
```

Comment:
does this mean get only 1000 Maps ?

Reply:
Yes, we have define a number here, the default is 20 in 1 page to response, which is obviously not enough.

Reply:
What if there are 1001 Maps? Are we iterating page?

Reply:
Updated here to have iteration request, per request will have 50 maps